test/exq_test.exs

```diff
@@ -362,6 +362,20 @@ defmodule ExqTest do
     assert_received {"short"}
   end
 
+  @tag :slow
+  test "configure worker shutdown time greater than 5000ms" do
+    Process.register(self(), :exqtest)
+    {:ok, sup} = Exq.start_link([shutdown_timeout: 6500])
+    {:ok, _} = Exq.enqueue(Exq, "default", ExqTest.SleepWorker, [7000, :long])
```

Comment:
Looks good, can we test smaller amount with this and just as long as `refute_received {"long"}` works it would still be a valid check?

Reply:
The default supervisor shutdown value is 5sec in OTP if not provided in the child spec.  Should we instead check that the child spec for the WorkerDrainer has the right shutdown value?

Reply:
I see, the only concern I had is adding extra time to the tests (10sec vs 16sec) - if that is an easy change, we can check the WorkerDrain instead.

Reply:
Ok sounds good. I will add a test for the Mode module and assert the child spec has the shutdown option.